### DISCUSSION TITLE
Need help finding channel links for WEAR, WKRG, WALA, and WPMI.

### DISCUSSION THREAD

**Discussion by KeiranGamingTV** [iptv-org/community#1163] [Q&A]
I need help finding channel links for these channels (All of the channel IDs can be found on [iptv-org.github.io](https://iptv-org.github.io)):

- **WEAR**
  - WEAR-DT1
  - WEAR-DT2
  - WEAR-DT3
- **WKRG**
  - WKRG-DT1
  - WKRG-DT2
  - WKRG-DT3
  - WKRG-DT4
- **WALA**
  - WALA-DT1
  - WALA-DT2
  - WALA-DT3
  - WALA-DT4
- **WPMI**
  - WPMI-DT1
  - WPMI-DT2
  - WPMI-DT3

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
